net/nheko/Portfile

```diff
@@ -4,26 +4,26 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.1
 PortGroup           qt5 1.0
-PortGroup           cxx11 1.1
 PortGroup           compiler_blacklist_versions 1.0
 
 #NOTE: requires C++14
```

Comment:
Please remove this comment; the newly added `compiler.cxx_standard 2014` makes it unnecessary.